Hero/Hero.swift

```diff
@@ -86,6 +86,20 @@ internal extension Hero {
   }
 }
 
+fileprivate extension Hero {
+  func closureProcessForHeroDelegate<T: UIViewController>(vc: T, closure: (HeroViewControllerDelegate)->()) {
+    switch vc {
+      case let fvc as HeroViewControllerDelegate:
+        closure(fvc)
+      case let fvc as UINavigationController:
```

Comment:
Here. It will only notify vc and not the topViewController if both implements HeroViewControllerDelegate.